### DISCUSSION TITLE
GAP8 multiple cores

### DISCUSSION THREAD

**Discussion by Sheng-Qian-LI** [bitcraze/discussions#1532] [Q&A]
Hello,

The GAP8 has 8+1 cores. When using the aideck-gap8-examples on GitHub, I think I am currently only using one of the cores(Because I can only control the frequency of the Fabric Controller, but not the Cluster .). 

Are there any examples that demonstrate how to enable and use all 8 cores simultaneously for computation? Or are there any examples in the aideck-gap8-examples that mention multi-core processing?

Thank you.

**Comment by gemenerik** [comment]
Check out [this workshop](https://www.youtube.com/watch?v=o9asYPHxEB4) which discusses parallelization of image manipulation
